setup.py

```diff
@@ -32,5 +32,8 @@
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
     ),
-    entry_points={"pulpcore.plugin": ["pulp_python = pulp_python:default_app_config", ]},
+    entry_points={
+        "pulpcore.plugin": ["pulp_python = pulp_python:default_app_config", ],
+        "pytest11": ["pulp_python = pulp_python.tests.functional"],
```

Comment:
We only make the fixtures a proper plugin if you have another python project that needs them. Typically for plugins they just want to use the fixtures in this repo so we don't make it a plugin.

Instead move your fixtures to the `conftest.py` directly and pytest will auto-find them for tests inside this project. See pulp_file for an example https://github.com/pulp/pulp_file/blob/main/pulp_file/tests/functional/conftest.py